Tensile/KernelWriter.py

```diff
@@ -585,7 +640,7 @@ def makeSubIterSchedule(self, kernel, localReadCode, iteration, pointerCode, wai
                 lgkmcnt = localWrites  # this only survives if writes are at the end
 
       lgkmcnt = min(lgkmcnt, 15)
-      waitCode.comment += " old=%u new=%u (Local write no wait)" % (waitCode.lgkmcnt, lgkmcnt)
+      waitCode.comment += " old=%u, new=%u newLW=%u newLR=%u" % (waitCode.lgkmcnt, lgkmcnt,localWrites,localReads)
```

Comment:
if not enableMatrixInstruction, localWrites/localreads will not be assigned.

Reply:
done